packages/secure_storage/amplify_secure_storage/android/build.gradle

```diff
@@ -54,7 +54,7 @@ dependencies {
     implementation 'androidx.security:security-crypto:1.0.0'
     // TODO(Jordan-Nelson): remove once security-crypto:1.1.0 is stable.
     // See https://github.com/aws-amplify/amplify-flutter/issues/2640
-    implementation 'com.google.crypto.tink:tink-android:1.8.0'
+    implementation 'com.google.crypto.tink:tink-android:1.9.0'
```

Comment:
Not sure if we want to ignore this one. I cannot find a change log for 1.9.0. We don't use tink directly. The only reason it is a dependency is to set the version to 1.8.0+.

Reply:
Should we set a range with a lower bound of 1.8.0? Is there a good reason to pin?